README.md

```diff
@@ -1,6 +1,9 @@
 # Besu Ethereum Client
- [![CircleCI](https://circleci.com/gh/hyperledger/besu/tree/main.svg?style=svg)](https://circleci.com/gh/hyperledger/besu/tree/main)
- [![Documentation Status](https://readthedocs.org/projects/hyperledger-besu/badge/?version=latest)](https://besu.hyperledger.org/en/latest/?badge=latest)
+
+ [![Acceptance Tests](https://github.com/hyperledger/besu/actions/workflows/acceptance-tests.yml/badge.svg?branch=main)](https://github.com/hyperledger/besu/actions/workflows/acceptance-tests.yml/badge.svg?branch=main)
+ [![Integration Tests](https://github.com/hyperledger/besu/actions/workflows/integration-tests.yml/badge.svg?branch=main)](https://github.com/hyperledger/besu/actions/workflows/integration-tests.yml/badge.svg?branch=main)
+ [![Container Security](https://github.com/hyperledger/besu/actions/workflows/container-security-scan.yml/badge.svg)](https://github.com/hyperledger/besu/actions/workflows/container-security-scan.yml/badge.svg)
```

Comment:
out of context idk how meaningful this label is - maybe it can be more specific? Docker Vulnerabilities Scanner?

Reply:
I can change the name of the GitHub action so that it show up with the same name.